addons/mass_mailing/wizard/mailing_mailing_test.py

```diff
@@ -68,7 +68,7 @@ def send_mail_test(self):
                 'auto_delete': False,  # they are manually deleted after notifying the document
                 'mail_server_id': mailing.mail_server_id.id,
                 'model': 'res.users',
-                'res_id': self.env.user.id,
+                'res_id': self.env['mailing.contact'].search([])[0].id,
```

Comment:
- Don't load in *ALL* contacts
- There could be *no* contacts
```suggestion
                'res_id': self.env['mailing.contact'].search([], limit=1)[:1].id,
```